test/fuzz/sql_fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+add_executable(sql_fuzzer
```

Comment:
Why are all the changes in this file needed in this patch?

Reply:
already removed